enhancements/ingress/ingress-dashboard.md

```diff
@@ -0,0 +1,199 @@
+
+In addition, product management has shown interest in providing and making more visible some cluster-wide statistics such as the number of routes and shards in use.
+
+This [request for enhancement](https://issues.redhat.com/browse/NE-1059) is also considered here, although it will not be entirely fulfilled as we don't intend here to create any new metric.
+
+### Goals
+
+1. Review and select a subset of the available metrics that should be included in the new dashboard. This should ideally be done with the help of the NetEdge team expertise and/or SREs.
+2. Design a new dashboard with these metrics.
+3. Update the Ingress Operator to deploy this dashboard, making it accessible for Cluster Monitoring consumption.
+
+### Non-Goals
+
+- This work is not intended to provide a comprehensive set of metrics all at once. Instead, the intent is to "start small", setting in place all the mechanisms in code, and iterate later based on feedback to add or amend elements in the dashboard without any code change.
+- This enhancement does not include any new metric creation or exposition: only already available metrics are considered. If discussions lead to consider the creation of new metrics, this could be the purpose of a follow-up enhancement.
+
+### User Stories
+
+> As a cluster administrator, I want to get a quick overview of general cluster statistics, such as the number of routes or shards in use.
+> As a cluster administrator, I want to get a quick insight in incoming traffic statistics, such as on latency and HTTP errors.
+
+## Proposal
+
+To make a new dashboard discoverable by Cluster Monitoring, a `ConfigMap` needs to be created in the namespace `openshift-config-managed`, containing a static dashboard definition in Grafana format (JSON). The dashboard datasource has to be Cluster Monitoring's Prometheus.
+
+The Ingress Operator is responsible for creating and reconciling this `ConfigMap`. We assume all metrics used in the dashboard are present unconditionally, which allows us to create a static dashboard unconditionally as well.
+In the event where conditional display should be introduced later (e.g. an alternative to HAProxy being implemented, resulting is different metrics to display), different approaches can be discussed: the dashboard could be dynamically amended via JSON manipulation; or several dashboards could be embedded and selectively installed.
+
+The content of the dashboard will be further discussed and refined, including during review and verification.
+As a starting point, we can focus on metrics mentioned in [NE-1059](https://issues.redhat.com/browse/NE-1059):
+
+- Number of external connections to each HAproxy instance/pod that could be grouped by client's source IP
```

Comment:
This metric, per client's source IP is likely not possible to have, ever, due to the metric cardinality that it would have. But just the number of connections, without grouping per client, should be doable

Reply:
Please update the EP to remove metrics that you are not planning to add.  In fact, the discussion from NE-1059 should not be a part of this proposal if you're not considering other input on what metrics to use and how to display, filter, or group them.

Reply:
Based on different comments, there is something that may be not clear.
We are not adding any metrics. We are only using existing metrics that are already collected by the monitoring stack and already reachable through Observe > Metrics page of the console.